crates/hdk/src/hash_path/path.rs

```diff
@@ -21,14 +21,22 @@ pub const DELIMITER: &str = ".";
 ///
 /// @todo - revisit whether there is a need/use-case for different link tags or entries
 /// See anchors implementation.
-pub const NAME: [u8; 8] = [0x68, 0x64, 0x6b, 0x2e, 0x70, 0x61, 0x74, 0x68];
+pub const DHT_PREFIX: [u8; 1] = [0x00];
```

Comment:
Need to update the docs here. Also, what's the purpose of there being a prefix now that it's not something recognizable?

Reply:
@maackle the prefix is here to give some kind of assurance that entry hashes don't collide with other app entries